modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi.adoc
+
+[id="csi-drivers-supported_{context}"]
+= CSI drivers supported by {product-title}
+
+{product-title} installs certain CSI drivers by default, giving users storage options that are not possible with in-tree volume plug-ins.
+
+To create CSI-provisioned persistent volumes that mount to these supported storage assets, {product-title} installs the necessary CSI driver Operator, the CSI driver, and the required storage class by default. For more details about the default namespace of the Operator and driver, see the documentation for the specific CSI Driver Operator.
+
+The following table describes the CSI drivers that are installed with {product-title} and which CSI features they support, such as volume snapshots, cloning, and resize.
+
+.Supported CSI drivers and features in {product-title}
+[cols=",^v,^v,^v,^v", width="100%",options="header"]
+|===
+|CSI driver  |CSI volume snapshots  |CSI cloning  |CSI resize |CSI ephemeral volumes
+
+|AWS EBS (Tech Preview) | ✅ | - | ✅ | -
+|Google Cloud Platform (GCP) persistent disk (PD) (Tech Preview)| ✅ | - | ✅ | -
+|OpenStack Cinder | ✅ | ✅ | ✅ | ✅
+|OpenStack Manila | ✅ | ✅ | ✅ | -
+|Red Hat Virtualization (oVirt) | ✅ | ✅ | ✅ | ✅
+|===
```

Comment:
@jsafrane - Could you PTAL at this table and let me know if it looks accurate to you in 4.7?

Reply:
AWS + GCE look good

I don't think anything supports in-line CSI ephemeral volumes - that's a feature dedicated to special CSI drivers.

Reply:
Thanks, @jsafrane . I've removed the CSI ephemeral volumes column from the table altogether.

Reply:
Cinder and Manila reviewed and approved by @Fedosin

Reply:
oVirt was reviewed by @bennyz and @rgolangh so we should be good with the table as it currently stands now.